tests/test_sklearn_tfidf_vectorizer_converter.py

```diff
@@ -577,6 +577,33 @@ def test_model_tfidf_vectorizer_nan(self):
         assert res.shape == (4, 9)
         assert numpy.isnan(res[0, 0])
 
+    @unittest.skipIf(
+        StrictVersion(onnx.__version__) <= StrictVersion("1.4.1"),
+        reason="Requires opset 9.")
+    def test_model_tfidf_vectorizer11_custom_vocabulary(self):
+        corpus = numpy.array([
+            "This is the first document.",
+            "This document is the second document.",
+            "And this is the third one.",
+            "Is this the first document?",
+        ]).reshape((4, 1))
+        vc = ["first", "second", "third", "document", "this"]
+        vect = TfidfVectorizer(ngram_range=(1, 1), norm=None, vocabulary=vc)
+        vect.fit(corpus.ravel())
+        model_onnx = convert_sklearn(vect, "TfidfVectorizer",
```

Comment:
Can you make sure that attribute `stop_words_` is missing?

Reply:
Yes, here is the description of stop_words_ attributes from sklearn:
`    stop_words_ : set
        Terms that were ignored because they either:
          - occurred in too many documents (`max_df`)
          - occurred in too few documents (`min_df`)
          - were cut off by feature selection (`max_features`).
        This is only available if no vocabulary was given.`
And this is what I got before modified:
`AttributeError: 'TfidfVectorizer' object has no attribute 'stop_words_'`